docs/usage/usage.md

```diff
@@ -355,7 +355,7 @@ nodeTemplate: # (to be specified only if the node capacity would be different fr
     memory: 50Gi
 diagnosticsProfile:
   enabled: true
-  # storageURI: <string>
+  # storageURI: https://<storage-account-name>.blob.core.windows.net/
```

Comment:
n.b.: That only works for the public instance

Reply:
should we add a hint in the section above?
In the end the user will notice when the machine creation fails...